env/env.go

```diff
@@ -8,6 +8,51 @@ import (
 
 // TODO: Move to `go-utils`
 
+// Repository ...
+type Repository interface {
```

Comment:
I worry this name is still too ambiguous. Also, a variable that refers to an instance is named `envRepository` and not just `repository`, so I would expect the interface to be named `EnvRepository` as well (or something similar).

Wdyt?

Reply:
golint fails for `env.EnvRepository`: `type name will be used as env.EnvRepository by other packages and that stutters; consider calling this Repository`

`Repository` in itself would be ambiguous, but you refer to it as `env.Repository`, which is not imo. When you assign it to a variable you can use `envRepository` as var name.